K480 Mantis Rifle

LeadOnTaste opened the pull request:
Добавлена снайперская винтовка "Скат" в компоновке "bullpup", обладающая повышенной мощностью по сравнению с "Касаткой", а также имеющая (кто бы мог подумать) прицел кратностью 1.5x для стрельбы на дальние дистанции. "Там сидела Мурка, в кожаной тужурке... ну а за спиною чёрный плазмаган!"

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->

LeadOnTaste committed:
K480 Mantis Rifle

Добавлена снайперская винтовка "Скат" в компоновке "bullpup", обладающая повышенной мощностью по сравнению с "Касаткой", а также имеющая (кто бы мог подумать) прицел кратностью 1.5x для стрельбы на дальние дистанции. "Там сидела Мурка, в кожаной тужурке..." ну а за спиною сука плазмаган!

LeadOnTaste committed:
Mantis Rebalance Patch

LeadOnTaste committed:
Личный "Скат"

LeadOnTaste committed:
Update k342.dm

HaCsO commented:
Мне больно видеть как из моей красивой и без того крутой пушечки делают гибган кастомку

HaCsO commented:
![image](https://user-images.githubusercontent.com/48949815/172009025-acdf3896-4624-4291-a6b0-793b8d2ae6bb.png)
На, умник

HaCsO commented:
Подозреваю что ты просто забыл инклюднуть файл .dm со своей кастомкой, и он не видит путь к предмету.

polerno commented:
Хуйлуша, а кто тебе сказал что тебе дали возможность получить кастомку? Никто, так что ПР запрещаю мёржить из-за нелегального пути получения кастомки

LeadOnTaste committed:
Revert "Личный "Скат""

This reverts commit 9a047c21e3b4eb788f5b7cbf88b55e5e1eeec938.

LeadOnTaste commented:
> Мне больно видеть как из моей красивой и без того крутой пушечки делают гибган кастомку

Не гибган, там урон едва ли выше оригинала. Револьвер блять больше ДПС даёт, чем этот ствол. И кастомка это отдельная херня, которая внешне отличается лишь описанием что там неебаца ястреб на прикладе выгравирован.

LeadOnTaste commented:
> Хуйлуша, а кто тебе сказал что тебе дали возможность получить кастомку? Никто, так что ПР запрещаю мёржить из-за нелегального пути получения кастомки

Либо заткни ебало либо разговаривай нормально в моём пулле.

polerno commented:
> > Хуйлуша, а кто тебе сказал что тебе дали возможность получить кастомку? Никто, так что ПР запрещаю мёржить из-за нелегального пути получения кастомки
> 
> Либо заткни ебало либо разговаривай нормально в моём пулле.

Дружок, ты кажется не понял. Съеби похорошему и удалил коммит о кастомке. А то это хуйня какая-то, ну или можешь просто съебать по дружески

LeadOnTaste commented:
> > > Хуйлуша, а кто тебе сказал что тебе дали возможность получить кастомку? Никто, так что ПР запрещаю мёржить из-за нелегального пути получения кастомки
> > 
> > 
> > Либо заткни ебало либо разговаривай нормально в моём пулле.
> 
> Дружок, ты кажется не понял. Съеби похорошему и удалил коммит о кастомке. А то это хуйня какая-то, ну или можешь просто съебать по дружески

Я сказал: либо нахуй с пляжу либо разговаривай нормально, пока у тебя гнилой базар - на твоё мнение клали хуй. Коммитом своим можешь подтереться, заодно подавившись его ревертом.

polerno commented:
> > > > Хуйлуша, а кто тебе сказал что тебе дали возможность получить кастомку? Никто, так что ПР запрещаю мёржить из-за нелегального пути получения кастомки
> > > 
> > > 
> > > Либо заткни ебало либо разговаривай нормально в моём пулле.
> > 
> > Дружок, ты кажется не понял. Съеби похорошему и удалил коммит о кастомке. А то это хуйня какая-то, ну или можешь просто съебать по дружески
> 
> Я сказал: либо нахуй с пляжу либо разговаривай нормально, пока у тебя гнилой базар - на твоё мнение клали хуй. Коммитом своим можешь подтереться, заодно подавившись его ревертом.

Мой коммит хотя бы люди просили, а твоё шизофреничное говно никто не просил и ты просто сделал. Хочешь чтобы я съебал? Закрой ПР и я съебу

LeadOnTaste commented:
> > > > > Хуйлуша, а кто тебе сказал что тебе дали возможность получить кастомку? Никто, так что ПР запрещаю мёржить из-за нелегального пути получения кастомки
> > > > 
> > > > 
> > > > Либо заткни ебало либо разговаривай нормально в моём пулле.
> > > 
> > > 
> > > Дружок, ты кажется не понял. Съеби похорошему и удалил коммит о кастомке. А то это хуйня какая-то, ну или можешь просто съебать по дружески
> > 
> > 
> > Я сказал: либо нахуй с пляжу либо разговаривай нормально, пока у тебя гнилой базар - на твоё мнение клали хуй. Коммитом своим можешь подтереться, заодно подавившись его ревертом.
> 
> Мой коммит хотя бы люди просили, а твоё шизофреничное говно никто не просил и ты просто сделал. Хочешь чтобы я съебал? Закрой ПР и я съебу

Я предложение закинул, при том с конкретным фундаментом под ним, предложение обсудили. Возражений не было, кастомку ревертнул, чтоб Иви не бухтела. А теперь прошу нахуй пройти, пока разговаривать не научишься нормально.

polerno commented:
> > > > > > Хуйлуша, а кто тебе сказал что тебе дали возможность получить кастомку? Никто, так что ПР запрещаю мёржить из-за нелегального пути получения кастомки
> > > > > 
> > > > > 
> > > > > Либо заткни ебало либо разговаривай нормально в моём пулле.
> > > > 
> > > > 
> > > > Дружок, ты кажется не понял. Съеби похорошему и удалил коммит о кастомке. А то это хуйня какая-то, ну или можешь просто съебать по дружески
> > > 
> > > 
> > > Я сказал: либо нахуй с пляжу либо разговаривай нормально, пока у тебя гнилой базар - на твоё мнение клали хуй. Коммитом своим можешь подтереться, заодно подавившись его ревертом.
> > 
> > Мой коммит хотя бы люди просили, а твоё шизофреничное говно никто не просил и ты просто сделал. Хочешь чтобы я съебал? Закрой ПР и я съебу
> 
> Я предложение закинул, при том с конкретным фундаментом под ним, предложение обсудили. Возражений не было, кастомку ревертнул, чтоб Иви не бухтела. А теперь прошу нахуй пройти, пока разговаривать не научишься нормально.

Ты первым насрал, я высказал недовольство как быдло. Так что съеби заткни ебало

LeadOnTaste commented:
> > > > > > > Хуйлуша, а кто тебе сказал что тебе дали возможность получить кастомку? Никто, так что ПР запрещаю мёржить из-за нелегального пути получения кастомки
> > > > > > 
> > > > > > 
> > > > > > Либо заткни ебало либо разговаривай нормально в моём пулле.
> > > > > 
> > > > > 
> > > > > Дружок, ты кажется не понял. Съеби похорошему и удалил коммит о кастомке. А то это хуйня какая-то, ну или можешь просто съебать по дружески
> > > > 
> > > > 
> > > > Я сказал: либо нахуй с пляжу либо разговаривай нормально, пока у тебя гнилой базар - на твоё мнение клали хуй. Коммитом своим можешь подтереться, заодно подавившись его ревертом.
> > > 
> > > 
> > > Мой коммит хотя бы люди просили, а твоё шизофреничное говно никто не просил и ты просто сделал. Хочешь чтобы я съебал? Закрой ПР и я съебу
> > 
> > 
> > Я предложение закинул, при том с конкретным фундаментом под ним, предложение обсудили. Возражений не было, кастомку ревертнул, чтоб Иви не бухтела. А теперь прошу нахуй пройти, пока разговаривать не научишься нормально.
> 
> Ты первым насрал, я высказал недовольство как быдло. Так что съеби заткни ебало

Вот в наливайке и будут тебя такие же быдланы слушать. А тут ты ровно нулевой хуйлан с такими манерами и слушать тебя никто не обязан.

polerno commented:
> > > > > > > > Хуйлуша, а кто тебе сказал что тебе дали возможность получить кастомку? Никто, так что ПР запрещаю мёржить из-за нелегального пути получения кастомки
> > > > > > > 
> > > > > > > 
> > > > > > > Либо заткни ебало либо разговаривай нормально в моём пулле.
> > > > > > 
> > > > > > 
> > > > > > Дружок, ты кажется не понял. Съеби похорошему и удалил коммит о кастомке. А то это хуйня какая-то, ну или можешь просто съебать по дружески
> > > > > 
> > > > > 
> > > > > Я сказал: либо нахуй с пляжу либо разговаривай нормально, пока у тебя гнилой базар - на твоё мнение клали хуй. Коммитом своим можешь подтереться, заодно подавившись его ревертом.
> > > > 
> > > > 
> > > > Мой коммит хотя бы люди просили, а твоё шизофреничное говно никто не просил и ты просто сделал. Хочешь чтобы я съебал? Закрой ПР и я съебу
> > > 
> > > 
> > > Я предложение закинул, при том с конкретным фундаментом под ним, предложение обсудили. Возражений не было, кастомку ревертнул, чтоб Иви не бухтела. А теперь прошу нахуй пройти, пока разговаривать не научишься нормально.
> > 
> > Ты первым насрал, я высказал недовольство как быдло. Так что съеби заткни ебало
> 
> Вот в наливайке и будут тебя такие же быдланы слушать. А тут ты ровно нулевой хуйлан с такими манерами и слушать тебя никто не обязан.

Нулевой хуйлан? Я как бы член команды Проксимы, так что слушать меня стоит. Так что не выбёбывайся и помолчи по хорошему

Evie7056 commented:
Добавь на базу ОБР, добавь уровни ресерча в оружейных технологиях. Пока что-то оно никак не добавляется в игру, как я вижу.

LeadOnTaste commented:
> Добавь на базу ОБР, добавь уровни ресерча в оружейных технологиях. Пока что-то оно никак не добавляется в игру, как я вижу.

Оно основано на коде Барракуды и потому тех. уровни те же. Маппить на базу ОБР я заебусь, торч и так долго компилируется.

Evie7056 commented:
Можно ставить предметы прямо через любой текстовой редактор. В теории. Сложно, но можно.

Ну или я не знаю, сунь тогда в карго, раз не можешь с ценой в районе 180 за две винтовки и двумя батареями.

LeadOnTaste commented:
> Можно ставить предметы прямо через любой текстовой редактор. В теории. Сложно, но можно.
> 
> Ну или я не знаю, сунь тогда в карго, раз не можешь с ценой в районе 180 за две винтовки и двумя батареями.

Эт можно.

LeadOnTaste committed:
Barracuda Spellcheck

Слегка подправил названия, чтоб пушки не выглядели как типичные стволы ГККшников. Также добавил несколько микроскинов на тему Vtuber-ов, которые могут рандомно появиться в шкафу вместо обычных винтовок. Нет, ПП по Гололайву я делать вам не буду, SMG Gura это не мой стиль уже. Ну блять, шутки шутками но я не виабушник.

LeadOnTaste committed:
Cargo Update

Добавил возможность заказа серийных (не кастомных) Барракуд, Касаток и Скатов в карго.

LeadOnTaste commented:
Короче, вот вам новые два коммита. Пасхальные скины по витуберам (шутка о том что РнД виабушники не стареет никогда) и апдейт карго

Evie7056 commented on code/datums/supplypacks/security.dm:
```diff
@@ -240,6 +240,30 @@
 	containername = "military rifle practice ammunition crate"
 	access = access_security
 
+/decl/hierarchy/supply_pack/security/barracuda
```
Ты убил из этого ящика батарейки. Теперь их нет в карго. Добавь по батарее на винтовку

LeadOnTaste replied:
я из этого ящика не трогал вроде ничего?

Evie7056 replied:
А где тогда батарейки в карго... Если честно я их не увидела, когда листала файл. Я их конечно все равно добавляю отдельным ящиком, но пусть пока будет в ящиках с оружием. И да, я погорячилась. Сделай две батарейки на пушку в ящиках. Средней

LeadOnTaste replied:
Готовы батарейки

Evie7056 commented on code/datums/supplypacks/security.dm:
```diff
@@ -240,6 +240,30 @@
 
+/decl/hierarchy/supply_pack/security/barracuda
+	name = "Weapons - Plasma assault rifles"
+	contains = list(/obj/item/gun/energy/k342 = 4)
```
Снизить до двух добавить батареи

Evie7056 commented on code/datums/supplypacks/security.dm:
```diff
@@ -240,6 +240,30 @@
+/decl/hierarchy/supply_pack/security/barracuda
+	name = "Weapons - Plasma assault rifles"
+	contains = list(/obj/item/gun/energy/k342 = 4)
+	cost = 120
```
110. Это старая цена ящика с плазмаганами, пусть она и останется

Evie7056 commented on code/datums/supplypacks/security.dm:
```diff
@@ -240,6 +240,30 @@
+/decl/hierarchy/supply_pack/security/orca
+	name = "Weapons - Plasma assault rifles (Expeditionary Corps)"
+	contains = list(/obj/item/gun/energy/k342/explo = 4)
+	cost = 150
```
130

LeadOnTaste replied:
Готово

Evie7056 commented on code/datums/supplypacks/security.dm:
```diff
@@ -240,6 +240,30 @@
+
+/decl/hierarchy/supply_pack/security/orca
+	name = "Weapons - Plasma assault rifles (Expeditionary Corps)"
+	contains = list(/obj/item/gun/energy/k342/explo = 4)
```
Также, снизить до двух, оставить батареи

Evie7056 commented on maps/torch/items/explo_shotgun.dm:
```diff
@@ -155,7 +155,8 @@
 //proxima code start
 /obj/item/gun/energy/k342/explo
-	desc = "K342 - Kasatka is the latest plasma weapon created by NanoTrasen. It can fire several types of charges: stunning, lethal and net-mode. This one designed specialy for expeditionary team"
+	name = "K342E Orca plasma assault rifle"
```
Никаких полных названий в имени оружия.
НИКАКИХ

Evie7056 replied:
Стоковый плазмаган имеет имя "плазмаган" и именно в таком стиле он и должен оставаться. Эта строчка не нужна

LeadOnTaste replied:
а шо не так

Evie7056 commented on maps/torch/items/explo_shotgun.dm:
```diff
@@ -155,7 +155,8 @@
 /obj/item/gun/energy/k342/explo
-	desc = "K342 - Kasatka is the latest plasma weapon created by NanoTrasen. It can fire several types of charges: stunning, lethal and net-mode. This one designed specialy for expeditionary team"
+	name = "K342E Orca plasma assault rifle"
+	desc = "K342E Orca is the latest plasma weapon created by NanoTrasen. This modification has been designed for Sol Central Government Expeditionary Corps and can fire several types of charges: stunning, lethal and net-mode."
```
Не помню, что бы я давала разрешение на переименовывание

LeadOnTaste replied:
да бля выглядит как будто гккшникам ствол делали, у нас же не руссиш марине

Evie7056 replied:
Оставь. Касатку в покое.

LeadOnTaste replied:
Нахуя. Писать. Названия. Транслитом.

Evie7056 commented on maps/torch/items/explo_shotgun.dm:
```diff
@@ -175,4 +176,17 @@
 /obj/item/gun/energy/k342/explo/prereg
 	authorized_modes = list(AUTHORIZED, AUTHORIZED, ALWAYS_AUTHORIZED)
+
+/obj/item/gun/energy/k342/explo/lily
```
Итак, это просто три лишние пушки с одинаковыми спрайтами, статами, но имеющие названия? Ужасно

Evie7056 replied:
Я все ещё вижу мусор.

Evie7056 replied:
И да. Названия. 
Не то что бы эти "рескины" вообще должны быть.

LeadOnTaste replied:
Описания ещё. Там по описаниям наклеечки-пинапы есть. Если уж форсить мемы про закидоны отделов, то форсить по полной. Начали с яблочной травы, закончили напитком из геншина.

Evie7056 commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -55,15 +63,31 @@
+
 /obj/item/projectile/plasma/stun
-	name = "stun plasma bolt"
+	name = "sniper stun plasma bolt"
```
Перемести изменение название имя в ребенка, а не меняй родителя

LeadOnTaste replied:
Поправлено

Evie7056 commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -78,8 +102,8 @@
 /obj/item/gun/energy/k342
-	name = "Plasma gun"
-	desc = "K342 - Barrakuda is the latest plasma weapon created by NanoTrasen. It can fire several types of charges: stunning, incendiary and lethal."
+	name = "K342 Barracuda plasma assault rifle"
```
Нет. Оставь название как было

Evie7056 commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -103,6 +127,22 @@
 	)
 
+/obj/item/gun/energy/k342/sniper
+	name = "K480 Mantis plasma sniper rifle"
```
Также, название должно быть чем-то вроде
name = "plasma sniper rifle"
Да, с маленькой.

Evie7056 commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -103,6 +127,22 @@
 
+/obj/item/gun/energy/k342/sniper
+	name = "K480 Mantis plasma sniper rifle"
+	desc = "K480 Mantis is the latest heavy plasma weapon created by NanoTrasen for SolGov snipers, capable to fire several types of charges: stunning, incendiary, and lethal bolts. Advanced magnetic constriction technology improves accuracy and firepower."
```
Мантис как-то не вписывается в тему плазмаганов, названных в честь рыб из семейства акулинвх или что-то вроде того, я не биолог, лол.
Пусть уж будет скат, это хотя бы рыба 🐟

LeadOnTaste replied:
Так Mantis по английски и будет скат

Evie7056 replied:
Хорошо, зря наехала

Evie7056 reviewed (changes requested):
Доделать и переделать

Evie7056 commented on maps/torch/items/explo_shotgun.dm:
```diff
@@ -134,7 +134,7 @@
 	return list(
 		/obj/item/cell/guncell/medium = 6,
-		/obj/item/gun/energy/k342/explo = 2
+		/obj/random/orca = 2
```
У нас этот ящик нигде не используется 😌
Он был у ПФ но я его удалила при реворке ЭК

LeadOnTaste committed:
Cargo Patch Ammo

LeadOnTaste committed:
Ёрш и Данго

Добавлены культовые напитки для пьющей и непьющей научки. На вкус заебись. Но с ершом не перебарщивайте, будет хуёво.

LeadOnTaste committed:
Убрал переименование, поправил стан-разряд

LeadOnTaste committed:
Update explo_shotgun.dm

LeadOnTaste committed:
Update security.dm

LeadOnTaste committed:
Update explo_shotgun.dm

Evie7056 commented on code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Drinks.dm:
```diff
@@ -600,15 +600,33 @@
 		S.handle_sugar(M,src,0.5)
 
+
+/datum/reagent/drink/dangomilk
```
Ебать, а что это тут забыло?

LeadOnTaste replied:
Эт я небольшую пасхалочку для учёных, рескин молочного коктейля (мем что учёные и ЭК виабушники) и ёрш для Чебурека (он же у нас любит пивдос)

Evie7056 commented on code/modules/reagents/Chemistry-Recipes.dm:
```diff
@@ -2194,6 +2194,12 @@
+/datum/chemical_reaction/dangomilk
+	name = "Dango Milk"
+	result = /datum/reagent/drink/dangomilk
+	required_reagents = list(/datum/reagent/drink/milk/cream = 1, /datum/reagent/nutriment/flour = 1, /datum/reagent/nutriment/rice = 1, /datum/reagent/drink/milk = 2)
```
Если добавляешь рецепты, то отрвзи это в пре. Иначе когда у нас будет своя Вики мы это забудем ещё до того как сделаем ее

Evie7056 commented on maps/torch/items/explo_shotgun.dm:
```diff
@@ -155,7 +155,7 @@
 //proxima code start
 /obj/item/gun/energy/k342/explo
-	name = "K342E Orca plasma assault rifle"
+	name = "K342E Orca plasma gun"
```
Да ты мыш. Никаких названий оружия в имени
name "plasma gun"

Evie7056 commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -102,7 +103,7 @@
 
 /obj/item/gun/energy/k342
-	name = "K342 Barracuda plasma assault rifle"
+	name = "K342 Barracuda plasma gun"
```
Да ты мыш х2
Никаких названий оружия в имени.

Evie7056 commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -128,7 +129,7 @@
 
 /obj/item/gun/energy/k342/sniper
-	name = "K480 Mantis plasma sniper rifle"
+	name = "K480 Mantis plasma gun"
```
Да ты мыш х3

HaCsO commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -55,6 +63,17 @@
+	fire_sound='proxima/sound/weapons/guns/vaporize.ogg'
+	armor_penetration = 75
+	agony = 15
+	damage = 90
```
Сколько ебать....

HaCsO replied:
гибган на связи

HaCsO replied:
![image](https://user-images.githubusercontent.com/48949815/172365191-3b7aeffe-bfb4-4007-8f92-4baff1b0e2e3.png)
антиматериалка би лайк...

LeadOnTaste replied:
ты сравниваешь хеви плазму снайперской версии с пулькой амрки

LeadOnTaste committed:
Update explo_shotgun.dm

LeadOnTaste committed:
Update k342.dm

HaCsO commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -103,6 +128,22 @@
+	init_firemodes = list(
+		list(mode_name="stun charge", projectile_type=/obj/item/projectile/plasma/stun/sniper, charge_cost=60, fire_delay=6, projectile_color=COLOR_YELLOW),
+		list(mode_name="plasma charge", projectile_type=/obj/item/projectile/plasma/sniper, charge_cost=60, fire_delay=6, projectile_color=COLOR_BLUE_LIGHT),
+		list(mode_name="heavy plasma charge", projectile_type=/obj/item/projectile/plasma/heavy/sniper, charge_cost=100, fire_delay=10, projectile_color=COLOR_RED)
```
Я считаю что это издевательство. с батарейкой на 600 юнитов можно выплюнуть 6 мегачарджев с 90 берна за 6 секунд. предлагаю сделать кд хотя бы секунд в 5

LeadOnTaste replied:
Ты сначала санкционируй эти мегачарджи.

LeadOnTaste replied:
А потом уже плюй их. И урон тут не сильно выше. Я эти статы в полтора раза вскинул.

LeadOnTaste committed:
Update k342.dm

HaCsO commented:
Ну полторы секунды заместо 1 это смешно и грустно

LeadOnTaste commented:
> Ну полторы секунды заместо 1 это смешно и грустно

Ну блять это не огнестрел чтоб у снайперки была ущербная скорострельность, тут 0,6 секунды хуярит любым другим режимом, а обычная вообще может в 0,4 секунды по пульке. Я вообще как марксманку этот ствол делал.

LeadOnTaste committed:
Update k342.dm

HaCsO commented:
а обычно баракуде то зачем а?.. Её же я делал, подбирал цифорки...

LeadOnTaste commented:
> а обычно баракуде то зачем а?.. Её же я делал, подбирал цифорки...

Ну дак там ораора скорострельность, это ни в какие ворота.

polerno commented:
> > а обычно баракуде то зачем а?.. Её же я делал, подбирал цифорки...
> 
> Ну дак там ораора скорострельность, это ни в какие ворота.

Лол, разве? Я сколько с неё стрелял. Она никогда быстро не херачила

LeadOnTaste commented:
> > > а обычно баракуде то зачем а?.. Её же я делал, подбирал цифорки...
> > 
> > 
> > Ну дак там ораора скорострельность, это ни в какие ворота.
> 
> Лол, разве? Я сколько с неё стрелял. Она никогда быстро не херачила

Видео Хаксо посмотри.

HaCsO commented:
> > > > а обычно баракуде то зачем а?.. Её же я делал, подбирал цифорки...
> > > 
> > > 
> > > Ну дак там ораора скорострельность, это ни в какие ворота.
> > 
> > 
> > Лол, разве? Я сколько с неё стрелял. Она никогда быстро не херачила
> 
> Видео Хаксо посмотри.

я это видео по приколу кинул, и поставил там на кд единичку

LeadOnTaste commented:
> > > > > а обычно баракуде то зачем а?.. Её же я делал, подбирал цифорки...
> > > > 
> > > > 
> > > > Ну дак там ораора скорострельность, это ни в какие ворота.
> > > 
> > > 
> > > Лол, разве? Я сколько с неё стрелял. Она никогда быстро не херачила
> > 
> > 
> > Видео Хаксо посмотри.
> 
> я это видео по приколу кинул, и поставил там на кд единичку

Ну ты пидор.

LeadOnTaste committed:
Update k342.dm

Evie7056 commented on maps/torch/items/explo_shotgun.dm:
```diff
@@ -178,15 +178,15 @@
 
 /obj/item/gun/energy/k342/explo/lily
-	name = "K342E 'Shy Lily' plasma gun"
+	name = "'Shy Lily' plasma rifle"
```
ПОЧфвафыфывлхфы вфвф
ыэвюфы
эвфы
вфывфыв
ПОЧЕМУ ЭТО ДО СИХ ПОР ПРЕ?????

LeadOnTaste replied:
А тебе не нравятся пасхалочки? Это так-то неотъемлемая часть игры

Evie7056 replied:
Мне кажется четыре комментария в одно место должны были показать.

LeadOnTaste replied:
Да чё те блять не так в визуале - пасхалочки и пасхалочки, укладываются в военный стиль к тому же. Я ж не делаю на половину Харона пин-ап.

Evie7056 replied:
Если я не вижу визуала - значит его нет 🐸

LeadOnTaste replied:
Ну как я тебе на трёх пикселях моледевочку жующую берцы нарисую? Как я тебе кицуне намалюю на них? Как я туда стесняшку-скреллочку вмещу?

Evie7056 replied:
Если графики нет - вообще никакой, то нафиг оно нужно. Это я пытаюсь донести уже хз сколько

Exapsters commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -55,6 +63,17 @@
+	fire_sound='proxima/sound/weapons/guns/vaporize.ogg'
+	armor_penetration = 75
+	agony = 15
+	damage = 80
```
Исправь урон на тридцать. Это слишком сильно.

LeadOnTaste replied:
Ты урон обычных барракуды и касатки видел в том же режиме?

Exapsters commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -55,6 +63,17 @@
+/obj/item/projectile/plasma/heavy/sniper
+	name = "heavy sniper plasma bolt"
+	fire_sound='proxima/sound/weapons/guns/vaporize.ogg'
+	armor_penetration = 75
```
Аналогично. Тут на также тридцать.

Evie7056 committed:
Ну меня шесть раз не надо спрашивать.Пять раз хватит

Exapsters commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -46,6 +46,14 @@
+/obj/item/projectile/plasma/sniper
+	name = "sniper plasma bolt"
+	fire_sound='proxima/sound/weapons/guns/vaporize.ogg'
+	armor_penetration = 40
```
Убери такую большую бронебойность. Максимум - пятнадцать.

Exapsters commented on proxima/code/game/items/weapons/gun/energy/k342.dm:
```diff
@@ -46,6 +46,14 @@
+	name = "sniper plasma bolt"
+	fire_sound='proxima/sound/weapons/guns/vaporize.ogg'
+	armor_penetration = 40
+	damage = 45
```
Урон двадцать ставь.

LeadOnTaste replied:
Да ёбаный рот, посмотри статы оригинала

Exapsters reviewed (changes requested):
Мораль такова, что нам не нужно очередное всеубивающее оружие. Исправляй. Желательно, также предоставь видео с тестов исправленной версии.

Evie7056 committed:
я насрала, соре

Evie7056 committed:
Update guns.dm

LeadOnTaste commented:
Закрываю этот шырзых на реструктуризацию, вышла параша с файлами.

Evie7056 commented:
Нахуя.
